### PR TITLE
External messaging rate limiting

### PR DESCRIPTION
**Pull request by leogrim**

### CODE REVIEW

**Comment by leogrim** [comment]
@stkem

**Comment by leogrim** [comment]
(I may have mixed up my commits ;)

File: kifi-backend/modules/eliza/app/com/keepit/eliza/model/NonUserThreadRepo.scala [hunk 1]
```diff
@@ -141,4 +143,10 @@ class NonUserThreadRepoImpl @Inject() (
     (for (row <- rows if row.accessToken===token) yield row).firstOption
   }
 
+  def getRecentRecipientsByUser(userId: Id[User], since: DateTime)(implicit session: RSession): Map[EmailAddressHolder, Int] = {
+    val relevantThreads = (for (row <- rows if row.createdBy === userId && row.createdAt > since) yield row)
```
Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/model/NonUserThreadRepo.scala:147]: We are going to need an index on `(created_by, createdAt)`

**Comment by stkem** [comment]
made one comment. lgtm otherwise.

**Comment by leogrim** [comment]
\+ index

**Pull request merged**